docs/configuration.md

```diff
@@ -0,0 +1,210 @@
+# Configuration
+
+## Project Wide
+
+You can turn on specific rules by toggling them in a
+`.template-lintrc.js` file at the base of your project, or at a custom relative
+path which may be identified using the CLI:
+
+```javascript
+module.exports = {
+  extends: 'recommended',
+
+  rules: {
+    'no-bare-strings': true,
+  },
+};
+```
+
+This extends from the builtin recommended configuration ([lib/config/recommended.js](lib/config/recommended.js)),
+and also enables the `no-bare-strings` rule (see [here](docs/rule/no-bare-strings.md)).
+
+Using this mechanism allows you to extend from the builtin, and modify specific rules as needed.
+
+Some rules also allow setting additional configuration, for example if you would like to configure
+some "bare strings" that are allowed you might have:
+
+```javascript
+module.exports = {
+  rules: {
+    'no-bare-strings': ['ZOMG THIS IS ALLOWED!!!!'],
+  },
+};
+```
+
+## Configuration Properties
+
+The following properties are allowed in the root of the `.template-lintrc.js` configuration file:
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`rules`</td>
+      <td>`Object`</td>
+      <td>
+        This is an object containing rule specific configuration (see details for each rule below).
+      </td>
+    </tr>
+    <tr>
+      <td>`extends`</td>
+      <td>`string|string[]`</td>
+      <td>
+        Either a string or an array of strings. Each string allows you to specify an internally
+        curated list of rules (we suggest `recommended` here).
+      </td>
+    </tr>
+    <tr>
+      <td>`pending`</td>
+      <td>`string[]`</td>
+      <td>
+        An array of module id&#39;s that are still pending. The goal of this array is to allow
+        incorporating template linting into an existing project without changing every single
+        template file. You can add all existing templates to this `pending` listing and slowly work
+        through them while at the same time ensuring that new templates added to the project pass
+        all defined rules. You can generate this list with the: `ember-template-lint *
+        --print-pending`.
+      </td>
+    </tr>
+    <tr>
+      <td>`ignore`</td>
+      <td>`string[]|glob[]`</td>
+      <td>
+        An array of module id&#39;s that are to be completely ignored. See [ignore
+        documentation](docs/ignore.md) for more details.
+      </td>
+    </tr>
+    <tr>
+      <td>`plugins`</td>
+      <td>`(string|Object)[]`</td>
+      <td>
+        An array of plugin objects or strings that resolve to files that export plugin objects. See
+        [plugin documentation](docs/plugins.md) for more details.
+      </td>
+    </tr>
+    <tr>
+      <td>`overrides`</td>
+      <td>`Array`</td>
```

Comment:
Looks like the backticks aren't displaying properly.

Reply:
Fixed.